src/bundlers/node/nodeJsLocalBundler.ts

```diff
@@ -30,7 +32,8 @@ const server = http.createServer((req, res) => {
   }
 });
 
-server.listen(port, hostname, () => {
+server.listen(port, () => {
+  console.log('Server running on port ' + port);
```

Comment:
This will look confusing for the user beucase there will be multiple servers running.

I propose to remove this console.log.

![image](https://github.com/Genez-io/genezio/assets/26139486/fe4e1a68-2766-4dd0-b90c-f5ce8baa8849)